src/app/f2f/controllers/photoIdSelection.js

```diff
@@ -88,7 +88,7 @@ class PhotoIdSelectionController extends BaseController {
         }
         case APP.PHOTO_ID_OPTIONS.NO_PHOTO_ID: {
           logger.info(
-            "photo-id-selection: user has selected No ID - redirecting to No ID page",
```

Comment:
is there a "No ID" page that needs to be removed?

Reply:
https://github.com/alphagov/di-ipv-cri-f2f-front/blob/main/src/lib/config.js#L35 

It looked to me like it's an external page outside of F2F but not 100%